R/collect.R

```diff
@@ -55,6 +55,24 @@
 #' [collect_extracts()] returns a tibble with columns for the resampling
 #' indicators, the location (preprocessor, model, etc.), and objects extracted
 #' from workflows via the `extract` argument to [control functions][control_grid()].
+#'
+#' @section Extracting with submodels:
+#'
+#' When making use of submodels, tune can generate predictions and calculate
+#' metrics for multiple model `.config`urations using only one model fit.
+#' However, this means that if a function was supplied to a
+#' [control function's][control_grid()] `extract` argument, tune can only
+#' execute that extraction on the one submodel that was fitted. As a result,
```

Comment:
I would avoid calling the fitted model a submodel. That conflicts with my understanding of "sub". 
I'd probably refer to the _fitted model_ and use _submodel_  for the unfitted model configs.

Reply:
Heard! [39674de](https://github.com/tidymodels/tune/pull/743/commits/39674dea0213a9660988d9d964c258674568dede).